src/pollyCLI/pollyintegration.cpp

```diff
@@ -326,19 +366,50 @@ QString PollyIntegration::createProjectOnPolly(QString projectname) {
     return runId;
 }
 
-/**
-* @brief This function parses the output of "createWorkflowRequest" command run from Qtprocess..
-* @param projectId
-* @return Return workflow request id for the pollyphi workflow..
-*/
-
 QString PollyIntegration::createWorkflowRequest(QString projectId){
     QString command2 = "createWorkflowRequest";
     QList<QByteArray> resultAndError = runQtProcess(command2, QStringList() << credFile << projectId);
     QString workflowRequestId = parseId(resultAndError.at(0));
     return workflowRequestId;
 }
 
+QString PollyIntegration::createRunRequest(QString componentId,
+                                           QString projectId)
+{
+    QString command = "createRunRequest";
+    QStringList arguments = QStringList() << credFile
+                                          << componentId
+                                          << projectId;
+    QList<QByteArray> resultAndError = runQtProcess(command, arguments);
+    QString runId = parseId(resultAndError.at(0));
+    return runId;
+}
+
+QString PollyIntegration::redirectionUiEndpoint(QString componentId,
+                                                QString runId)
+{
+    QString command = "getEndpointForRuns";
+    QStringList arguments = QStringList() << credFile;
+    QList<QByteArray> resultAndError = runQtProcess(command, arguments);
```

Comment:
Can we shift it to either of the two ? I prefer the latter one 
```
QStringList arguments(credFile);
QList<QByteArray> resultAndError = runQtProcess(command, arguments);
```
or 

```
QList<QByteArray> resultAndError = runQtProcess(command, QStringList(credFile));
```

Reply:
Will do.